ydb/library/yql/udfs/common/python/python3_small/test/cases/Annotations.sql

```diff
@@ -0,0 +1,67 @@
+--sanitizer ignore memory
```

Comment:
Disable already broken checks. (They had not worked in Arcadia since the beginning of the testing history)

Reply:
Currently, ``--sanitizer ignore memory`` is equivalent to ``--sanitizer ignore``.
I suggest to fix it in https://github.com/ydb-platform/ydb/pull/4153

Reply:
In general, failing tests should be fixed, but I suggest doing it separately from moving code in the OS.